### PR TITLE
Update for Scarab v3.4.0

### PR DESCRIPTION
**Pull request by nsoblath**
Package builder modifications

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -36,7 +34,7 @@ set ( Dripline_PYTHON_THROW_REPLY_KEYWORD "ThrowReply" CACHE STRING "Keyword use
 set( Dripline_MAX_PAYLOAD_SIZE "10000" CACHE STRING "Maximum payload size (bytes)" )
 
 # always use C++11
-set_option( USE_CPP11 TRUE )
+set( CMAKE_CXX_STANDARD 11 )
```
Comment (laroque) [inline CMakeLists.txt:37]: I have no reason to doubt that this is right, but hadn't there been discussion of moving to a newer standard? maybe 13?

**Comment by laroque** [comment]
Only concern is the failed build. I assume that this isn't using `buildx`, which maybe will resolve it?

File: CMakeLists.txt (hunk 1 above)
Comment (nsoblath) [reply CMakeLists.txt:37]: There's nothing wrong with using C++14, 17 or 20 for anything that uses dl-cpp.  But there's also no need for it for this particular package.  This is more of a minimum version.  There's not really a way of enforcing the minimum version aspect of it unless I were to put in much more complicated logic, which doesn't seem worth it.  Instead the build would just fail.

**Comment by nsoblath** [comment]
Looking at the failed build, I don't really understand why it didn't work.  I haven't seen this failure mode with the buildx version.

**Comment by nsoblath** [comment]
Interestingly, the last build worked (build 575).  As mentioned in #68, there are now interdependencies between these two PRs, in that #68 needs changes in this PR to pass the validation checks on Travis.  I propose that if everything else looks okay here, we merge the PRs and then validate everything on develop.

**Pull request merged**